src/components/navbar.tsx

```diff
@@ -1,14 +1,12 @@
 import Link from "next/link";
 
-function Navbar() {
+export function Navbar() {
   return (
     <div className="flex-col space-x-4 text-blue-800">
       <Link href="/">Strona Główna</Link>
       <Link href="/artists">Artyści</Link>
       <Link href="/map">Mapa Wydarzenia</Link>
-      <Link href="/posts-fb">Aktualności</Link>
+      <Link href="/postsFb">Aktualności</Link>
```

Comment:
Tu coś poszło nie tak
```suggestion
      <Link href="/posts-fb">Aktualności</Link>
```

Reply:
w tym drugim branchu i tak zmieniłem na `/news`, więc się zmieni tak czy siak